File: sorting-the-sentence/sorting-the-sentence.js
/**
 * @param {string} s
 * @return {string}
 */
var sortSentence = function(s) {
  s = s.split(' ');
  let temp = [];
  for (let i = 0; i < s.length; i++) {
    let length = s[i].length - 1;
    temp[s[i][length] - 1] = s[i].slice(0, length);
  }
  return temp.join(' ');
};

File: sorting-the-sentence/sorting-the-sentence.py
class Solution:
    def sortSentence(self, s: str) -> str:
        clone = s.split(' ')
        res = clone.copy()
        
        for i in clone:
          word = i[0 : len(i) - 1]
          position = i[len(i) - 1 : len(i)]
          res[int(position) - 1] = word
        return ' '.join(res)